recipes/dj-static/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set package = "dj-static" %}
```

Comment:
What is the purpose?  Not critical, just curious.  Does it make it easier to create other recipes by copy/paste?  Just nicer to look at?

Version templating I understand, since it changes, but I don't see the immediate value of this or the md5.  I'm wide open to being enlightened.